components/library/gtk3-nocsd/Makefile

```diff
@@ -13,11 +13,13 @@
 # Copyright 2017 Alexander Pyhalov
 #
 
+BUILD_BITS=			64
```

Comment:
We should check devhelp look and feel. If it uses csd, should recompile it 64-bit.

Reply:
Yes, we will have recursive effects. That's the reason I marked this PR with (WIP).
Also there are more problems I encountered:
pygobject-3 doesn't build 32-bit anymore, totem doesn't build anymore (Cannot find required GStreamer-0.10 plugin 'playbin2'. It should be part of gst-plugins-base. Please install it.)

Reply:
pygobject-3 is expected, likely can just drop 32-bit.
totem builds here...

Reply:
It built last time I tried (mid-May probably). But today 2 build machines have problems:
...
checking for gstreamer-plugins-base-0.10... yes
checking for gstreamer-0.10 >= 0.10.30 gstreamer-base-0.10 >= 0.10.30 gstreamer-plugins-base-0.10 >= 0.10.30 gstreamer-tag-0.10 >= 0.10.30... yes
/oibuild/oi-userland/components/multimedia/totem/totem-bfc537c4cdcd51e747611a37fc3583d26e4d634f/configure: line 14761: -x: command not found
checking GStreamer 0.10 inspection tool... yes
checking GStreamer 0.10 playbin2 plugin... no
configure: error: 
			Cannot find required GStreamer-0.10 plugin 'playbin2'.
			It should be part of gst-plugins-base. Please install it.
		
gmake: *** [/oibuild/oi-userland/make-rules/configure.mk:175: /oibuild/oi-userland/components/multimedia/totem/build/amd64/.configured] Error 1

I am working on dropping 32-bit for pygobject-3.